### PR TITLE
Upload files in chunks

### PR DESCRIPTION
**Pull request by JMSS-Unknown**
Uploads files in chunks to avoid failed uploads in Chrome.

### CODE REVIEW

File: src/utils/chunkUpload.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import api from '../api'
+
+const CHUNK_SIZE = 100 * 1024 * 1024
```
Comment (Teg64) [inline src/utils/chunkUpload.ts:7]: Please add a comment briefly describing why this is the chunk size

**Review by Teg64 — CHANGES_REQUESTED** [review]
All looks good. Please just add some small documentation on the chunk size

File: src/utils/chunkUpload.ts (hunk 1 above)
Comment (JMSS-Unknown) [reply src/utils/chunkUpload.ts:7]: Done

**Comment by JMSS-Unknown** [comment]
Addressed the comments, please have a look.

**Review by Teg64 — APPROVED** [review]
Looks good!

**Pull request merged**